gcloud/datastore/demo/demo.py

```diff
@@ -95,11 +95,11 @@
 datastore.delete([key, key2])
 
 # To rollback a transaction, just call .rollback()
-with datastore.Transaction() as t:
+with datastore.Transaction() as xact:
     key = datastore.Key('Thing', 'another')
     thing = datastore.Entity(key)
-    thing.save()
-    t.rollback()
+    xact.put(thing)
+    xact.rollback()
```

Comment:
I suppose the ongoing existence of this line contradicts what I said above just a wee bit.